COPYRIGHT

```diff
@@ -0,0 +1 @@
+Copyright 2019 Association of Universities for Research in Astronomy
```

Comment:
`2019` -> `2024`.

Reply:
Updated in the commit: [c90119c](https://github.com/lsst-ts/ts_rotgui/pull/1/commits/c90119c88c04df93ef3f0d07d8af7cfa121ec2ec). Thanks!